libzfs.pyx

```diff
@@ -2626,6 +2626,11 @@ cdef class ZPoolScrub(object):
             if self.stats != NULL:
                 return self.stats.pss_pass_issued
 
+    property bytes_skipped:
+        def __get__(self):
+            if self.stats != NULL:
+                return self.stats.pss_skipped
```

Comment:
This doesn't work since `self.status` is a pointer to `pool_scan_stat_t` object and python doesn't know how to convert it to an object that can be represented in python.

Reply:
That's what ChatGPT says. I think the real issue is that `pss_skipped` is not part of that structure as our docker image uses outdated ZFS.

Reply:
I'm not sure how ChatGPT has anything to do with this

Reply:
@themylogin the image was updated about 10 days ago https://hub.docker.com/r/ixsystems/zfs, how recent is this change ?

Reply:
Saw the related PR, makes sense :P nvm above

Reply:
> @themylogin the image was updated about 10 days ago https://hub.docker.com/r/ixsystems/zfs, how recent is this change ?

Looks like `pss_skipped` is a Mav improvement in ZFS 2.2.